packages/common/src/utils/zod-builder/zod-builder.ts

```diff
@@ -0,0 +1,6 @@
+import { ZodTypeAny } from 'zod';
+
+export const zodBuilder =
+  <TType>() =>
+  <TSchema extends ZodTypeAny & { _output: TType }>(schema: TSchema) =>
+    schema;
```

Comment:
would like to have an explanation why is needed ?

i already would say that lets try to use [ajv-ts](https://github.com/vitalics/ajv-ts) instead keep working on advanced zod schema builders